src/NodeVisitor/FunctionCallScoperNodeVisitor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the humbug/php-scoper package.
+ *
+ * Copyright (c) 2017 Théo FIDRY <theo.fidry@gmail.com>,
+ *                    Pádraic Brady <padraic.brady@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Humbug\PhpScoper\NodeVisitor;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+
+final class FunctionCallScoperNodeVisitor extends NodeVisitorAbstract
+{
+    /**
+     * @var string
+     */
+    private $prefix;
+
+    /**
+     * Function which first parameter should be prefixed.
+     *
+     * @var array
```

Comment:
`string[]`, but to be fair, for both `$prefix` and `$functions`, as they are defined just bellow with the typehint I wouldn't add the phpdoc here:

```php
private $prefix;
private $functions;
```